test/parallel/test-cli-nodeopt.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+const common = require('../common');
+if (process.config.variables.node_without_node_options)
+  return common.skip('missing NODE_OPTIONS support');
+
+// Test options specified by env variable.
+
+const assert = require('assert');
+const exec = require('child_process').execFile;
+
```

Comment:
The test should likely include all whitelisted options to ensure that those do not cause the process to error out.